packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -171,10 +171,16 @@ custom domain to the `$default` stage of the API.
 ```ts
 const certArn = 'arn:aws:acm:us-east-1:111111111111:certificate';
 const domainName = 'example.com';
-
-const dn = new DomainName(stack, 'DN', {
+const domainNameConfigurations = new Array<DomainNameConfiguration>();
+const dnConfig: DomainNameConfiguration = {
+  certificate: Certificate.fromCertificateArn(stack, 'cert', certArn),
+  endpointType: EndpointType.REGIONAL,
```

Comment:
do we need the trailing comma?

also, can you add the securityPolicy to the DomainNameConfiguration too? It's good for any sample to use the ideal version we want customers to use, which is TLS 1.2

Reply:
yup, we do. I tried without it, the compilation fails.